src/services/account.validation.service.ts

```diff
@@ -97,20 +102,20 @@ export function mapResponseType(
             case "OK":
                 return {
                     status: "success",
-                    fileId: accountValidatorResponse.fileId,
+                    ...baseResult
                 };
             case "FAILED":
                 return {
                     status: "failure",
                     reasons: result.errorMessages,
-                    fileId: accountValidatorResponse.fileId,
+                    ...baseResult
                 };
     }
 }
 
 const idMap = {
-    "success.xhtml": "1f105de5-1ace-46df-84e4-15bfb9d1411a",
-    "failure_duplicate_facts.xhtml": "2a089d94-785e-42a9-97cf-c85c26ee83ca",
+    "success.xhtml": "50023f7a-8e66-461e-a9c8-230e6d8b598a",
```

Comment:
I believe this is a temporary idMap being created until another JIRA story which deals with Web calling File Transfer Service is done, isn't?

Reply:
Yes this is temporary until [BI-12387](https://companieshouse.atlassian.net/browse/BI-12387) merged. The ID's are changed in this PR because the previous ones were uploaded by an old version of the `file-transfer-service` which did not add the file name as metadata and hence could not return them. These new ID's are files which have the file name metadata. 
They will be remove in the furture.